.github/workflows/container.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           # Fedora latest stable version
           - {distro: fedora, image: 'fedora:latest'}
           # Fedora development version
-          - {distro: fedora, image: 'fedora:rawhide'}
+          - {distro: fedora, image: 'fedora:rawhide', ssl_dir: '/tmp/mysql2'}
```

Comment:
I added the `ssl_dir` option to test this feature.
You can see the `my.cnf` is set up with the environment variable.
https://github.com/brianmario/mysql2/actions/runs/3758719509/jobs/6387395753#step:4:85

You can also see the SSL tests are not skipped. If the pem files don't exist, the tests are skipped.
https://github.com/brianmario/mysql2/actions/runs/3758719509/jobs/6387395753#step:4:664